docs/Agent/user-guide/man_page.md

```diff
@@ -1 +1,469 @@
-# Man Page
+# Man pages
+ 
+**Contents**
+
+#### Performance tool management commands
+-  [pbench-clear-results](#pbench-clear-results)
+-  [pbench-clear-tools](#pbench-clear-tools)
+-  [pbench-list-tools](#pbench-list-tools)
+-  [pbench-list-triggers](#pbench-list-triggers)
+-  [pbench-register-tool](#pbench-register-tool)
+-  [pbench-register-tool-set](#pbench-register-tool-set)
+-  [pbench-register-tool-trigger](#pbench-register-tool-trigger)
+#### Benchmark commands
+-  [pbench-user-benchmark](#pbench-user-benchmark)
+#### Upload to Pbench Server
+###### Pbench Server 0.69
+- [pbench-move-result](#pbench-move-result)
+- [pbench-copy-results](#pbench-copy-results) 
+###### Pbench Server 1.0
+-  [pbench-results-move](pbench-results-move)
+
+
+---
+
+##### pbench-clear-results
+
+---
+
+**NAME**  
+`pbench-clear-results`:
```

Comment:
The trailing `:` is distracting; and now *this* entry has *only* the name while all the others still have "name: short description". We have `**DESCRIPTION**` below, and since these aren't section headers I'm not sure having a description on the same line is really of any use ... but at least they should all follow the same pattern.